### PR TITLE
[Feature] Image Previews Implementation Across Document Tables

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 The PR includes implementations for image previews across document tables. Screenshot:

Screenshot 2023-09-01 at 18 31 32

Let me know your thoughts.

### CODE REVIEW

**Comment by turbo124** [comment]
Looks great! Thanks.

File: src/components/DocumentsTable.tsx [hunk 1]
```diff
@@ -107,6 +121,39 @@ export function DocumentsTable(props: Props) {
...
 
+  useEffect(() => {
+    if (reactSettings.show_document_preview) {
+      props.documents.forEach(async ({ id, hash, type }) => {
+        const alreadyExist = documentsUrls.find(
+          ({ documentId }) => documentId === id
+        );
+
+        if (!alreadyExist && (type === 'png' || type === 'jpg')) {
+          await queryClient.fetchQuery(
```
Comment (beganovich) [inline src/components/DocumentsTable.tsx:132]: Looks like we're missing caching here. Can we cache infinitely?

['documents', hash]

File: src/common/hooks/useReactSettings.ts [hunk 2]
```diff
@@ -67,6 +68,7 @@ export function useReactSettings() {
   const settings: ReactSettings = {
     show_pdf_preview: true,
     react_notification_link: true,
+    show_document_preview: true,
```
Comment (beganovich) [inline src/common/hooks/useReactSettings.ts:71]: We definitely want this _false_ by default.

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, few changes needed.

File: src/common/hooks/useReactSettings.ts (hunk 2 above)
Comment (Civolilah) [reply src/common/hooks/useReactSettings.ts:71]: Sure, changed!

**Pull request merged**